frontend/public/docs/mssc-ches-merge-example.txt

```diff
@@ -0,0 +1,16 @@
+<h2>{{greeting}} {{scope}}</h2>
```

Comment:
These intended to be different case (Ie "greeting" vs "Greeting") as the example above?

Reply:
greeting (variable name) is how Greeting (column name) is transformed, so that's what the template takes.